### PR TITLE
Implement __abs__ in Series, Index and DataFrame

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR implements `__abs__` in Koalas Series, Index and DataFrame.

Pandas:

```python
>>> import pandas as pd
>>> abs(pd.Series([-1, -2, -3]))
0    1
1    2
2    3
dtype: int64
>>> import numpy as np
>>> np.abs(pd.Series([-1, -2, -3]))
0    1
1    2
2    3
dtype: int64
```

Koalas:

```python
>>> from databricks import koalas as ks
>>> abs(ks.Series([-1, -2, -3]))
0    1
1    2
2    3
Name: 0, dtype: int64
>>> import numpy as np
>>> np.abs(ks.Series([-1, -2, -3]))
0    1
1    2
2    3
Name: 0, dtype: int64
```

### CODE REVIEW

File: databricks/koalas/namespace.py [hunk 1]
```diff
@@ -86,7 +87,9 @@
 ]
 
 
-def from_pandas(pobj: Union["pd.DataFrame", "pd.Series"]) -> Union["Series", "DataFrame"]:
+def from_pandas(
+    pobj: Union["pd.DataFrame", "pd.Series", "pd.Index"]
+) -> Union["Series", "DataFrame", "Index"]:
     """Create a Koalas DataFrame or Series from a pandas DataFrame or Series.
```
Comment (itholic) [inline databricks/koalas/namespace.py:93]: How about also add a description about Index to the docstring ? 
Something like `Create a Koalas DataFrame, Series or an Index  from a pandas DataFrame, Series or an Index.`

File: databricks/koalas/tests/test_indexes.py [hunk 2]
```diff
@@ -1310,3 +1310,15 @@ def test_index_get_level_number(self):
 
         for lv, output in zip(level_names, outputs):
             self.assertEqual(output, kdf.index._get_level_number(lv))
+
+    def test_abs(self):
+        pdf = pd.DataFrame({"a": [-2, -1, 0, 1]}, index=[-2, -1, 0, 1])
+        pidx = pdf.index
```
Comment (itholic) [inline databricks/koalas/tests/test_indexes.py:1316]: nit question: Why don't we just create an index directly `pd.index([-2, -1, 0, 1])` ?
(Maybe you wanted to test for `Index` from `DataFrame` would work properly ? 😃 )

**Comment by itholic** [comment]
LGTM except several nit questions.

File: databricks/koalas/tests/test_indexes.py [hunk 3]
```diff
@@ -1310,3 +1310,14 @@ def test_index_get_level_number(self):
...
             self.assertEqual(output, kdf.index._get_level_number(lv))
+
+    def test_abs(self):
+        pidx = pd.Index([-2, -1, 0, 1])
+        kidx = ks.from_pandas(pidx)
+
+        self.assert_eq(abs(pidx), abs(kidx))
+        self.assert_eq(np.abs(pidx), np.abs(kidx))
+
+        kidx = ks.from_pandas(pd.MultiIndex.from_tuples([(1, 2)], names=["level1", "level2"]))
```
Comment (ueshin) [inline databricks/koalas/tests/test_indexes.py:1321]: nit: we have `ks.MultiIndex.from_tuples`.

**Pull request merged**